docs/user/config-file/v2.rst

```diff
@@ -228,20 +227,6 @@ With the previous settings, Read the Docs will execute the next commands:
 
    pip install .[docs]
 
-python.system_packages
```

Comment:
We should at least note a formal deprecation of this somewhere, so users who are using it can find that mention -- otherwise it's just not mentioned in the docs at all?